src/compiler/checker.ts

```diff
@@ -1938,6 +1938,9 @@ namespace ts {
                         const moduleName = getFullyQualifiedName(moduleSymbol);
                         const declarationName = declarationNameToString(name);
                         const suggestion = getSuggestedSymbolForNonexistentModule(name, targetSymbol);
+                        const defaultSymbolMatch = moduleName.includes("default") ?

```

Comment:
`includes` is an ES6 library feature, so please use one of the helpers in `core.ts`